sormas-api/src/main/resources/captions.properties

```diff
@@ -41,6 +41,7 @@ total=Total
 unknown=Unknown
 creationDate=Creation date
 notAvailableShort=NA
+numberOfCharacters=Number of characters %d / %d
```

Comment:
I'd add a colon after "Number of characters"

Reply:
Done